patches/tModLoader/Terraria/ModLoader/DamageClass.cs

```diff
@@ -5,8 +5,8 @@ namespace Terraria.ModLoader
 {
 	public abstract class DamageClass : ModType
 	{
+		public static DamageClass Default { get; private set; } = new DefaultDamageClass();
```

Comment:
For support sanity's sake, please add "[Breaking Changes]" to the title so people have a higher chance of clueing in.

Reply:
done

Reply:
(Resolved)